wasabi/printer.py

```diff
@@ -187,10 +247,10 @@ def divider(self, text="", char="=", show=True, icon=None):
             return text
         print(text)
 
-    def table(self, data, **kwargs):
+    def table(self, data: Union[Collection, Dict], **kwargs):
         """Print data as a table.
 
-        data (iterable / dict): The data to render. Either a list of lists
+        data (Collection / Dict): The data to render. Either a list of lists
```

Comment:
The `Iterable` type causes an error here:

```
# The `data` variable is passed on to wasabi.table
wasabi/tables.py:69: error: Argument 1 to "len" has incompatible type "Iterable[Any]"; expected "Sized"
```

I decided to use `Collection` since I don't think there's no need for this data to be ordered.